cekit/builders/docker_builder.py

```diff
@@ -131,6 +136,27 @@ def build(self, build_args=None):
             logger.info("Image built and available under following tags: %s"
                         % ", ".join(self._tags))
 
+        except ConnectionError as ex:
+            exception_chain = traceback.format_exc()
+            logger.debug("Caught ConnectionError attempting to communicate with Docker ", exc_info=1)
+
+            if 'PermissionError' in exception_chain:
+                message = "Unable to contact docker daemon. Is it correctly setup?\n" \
+                          "See https://developer.fedoraproject.org/tools/docker/docker-installation.html and " \
+                          "http://www.projectatomic.io/blog/2015/08/why-we-dont-let-non-root-users-run-docker-in-centos-fedora-or-rhel"
+            elif 'FileNotFoundError' in exception_chain:
+                message = "Unable to contact docker daemon. Is it started?"
+            else:
+                message = "Unknown ConnectionError from docker ; is the daemon started and correctly setup?"
+
+            if sys.version_info.major == 3:
+                # Work-around for python 2 / 3 code - replicate exception(...) from None
+                cekit = CekitError(message, ex)
```

Comment:
I would probably rather not use `cekit` as the variable name here. Use `cekit_error` or anything else.

Reply:
Ok no problem; thanks!